proto/cork/v1/query.proto

```diff
@@ -28,6 +28,8 @@ service Query {
   rpc QueryCellarIDs(QueryCellarIDsRequest) returns (QueryCellarIDsResponse) {
     option (google.api.http).get = "/sommelier/cork/v1/cellar_ids";
   }
+  // QueryScheduledCorks returns all scheduled corks
+//  rpc QueryScheduledCorks
```

Comment:
Looks like the implementation is missing here.

Reply:
I think this information is broadly useful to be able to provide to people who want to query the chain for sure.